client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SnapshotClientDocumentationIT.java

```diff
@@ -367,6 +369,74 @@ public void onFailure(Exception e) {
         }
     }
 
+    public void testSnapshotSnapshotsStatus() throws IOException {
+        RestHighLevelClient client = highLevelClient();
+        createTestRepositories();
+        createTestSnapshots();
+
+        // tag::snapshots-status-request
+        SnapshotsStatusRequest request = new SnapshotsStatusRequest();
+        // end::snapshots-status-request
+
+        // tag::snapshots-status-request-repository
+        request.repository(repositoryName); // <1>
+        // end::snapshots-status-request-repository
+        // tag::snapshots-status-request-snapshots
+        String [] snapshots = new String[] {snapshotName};
+        request.snapshots(snapshots); // <1>
+        // end::snapshots-status-request-snapshots
+        // tag::snapshots-status-request-ignoreUnavailable
+        request.ignoreUnavailable(true); // <1>
+        // end::snapshots-status-request-ignoreUnavailable
```

Comment:
These are not actually rendered since the doc markup does not have them included in it.

Reply:
LOL disregard clicked the wrong link!